samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -111,6 +111,8 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       rtl={currentRtl}
       callInvitationUrl={callInvitationUrl}
       formFactor={isMobileSession ? 'mobile' : 'desktop'}
+      /* @conditional-compile-remove(call-readiness) */
+      options={{ callReadinessOptedIn: true }}
```

Comment:
Should this be set to true? Wouldn't the prop be passed down?
The Calling Sample and CallWithChat sample would pass down the prop to set this value

Reply:
callReadinessOptedIn is a callcomposite option, I believe it should be set directly on the callcomposite? similar to what we did with rooms and devicepermission options

Reply:
fixed